openapi/integrator/token-requestor/bankaxept.yaml

```diff
@@ -63,6 +63,34 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks:
+    get:
+      summary: Eligible bank identifiers
+      description: From the input of comma-separated list of bank identifiers, return the ones that are eligible for enrolment with ePayment.
```

Comment:
The format of the parameter should not be repeated in the description, that is redundant and poorly maintainable. How about something like `Returns the subset of the inputted list of banks that are eligible for enrolment.` instead?

Reply:
I don't see why not, but sure, and I'll use slightly better English. :D